Android/PowerUpApp/PowerUpApp/res/layout/completed_game.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
```

Comment:
This attribute doesn't apply to RelativeLayout. Use LinearLayout whenever it applies